pkgs/top-level/all-packages.nix

```diff
@@ -32247,7 +32247,7 @@ with pkgs;
   };
 
   neovimUtils = callPackage ../applications/editors/neovim/utils.nix {
-    inherit (lua51Packages) buildLuarocksPackage;
+    lua = lua5_1;
```

Comment:
```suggestion
    lua = luajit;
```

Reply:
I think it should actually be neovim-unwrapped.lua but it just changes which version of luarocks you are going to use to install the packages so I would rather not change it for now to avoid breakage. What matters  is the version of lua run by neovim, which is actually luajit so perfs are still  the best